modules/gitops-configuring-argo-cd-oidc.adoc

```diff
@@ -0,0 +1,77 @@
+// Module is included in the following assemblies:
+//
+// * 
+
+[id="configuring-argo-cd-oidc_{context}"]
+= Configuring Argo CD OIDC
+
+.Prerequisites
+
+* You have obtained your client secret. 
+
+.Procedure
+
+. Store the client secret you generated earlier in the argocd secret argocd-secret.
+
+. Encode the client secret in base64:
++
+[source,terminal]
+----
+$ echo -n '83083958-8ec6-47b0-a411-a8c55381fbd2' | base64
+----
+
+. Edit the secret and add the base64 value to an `oidc.keycloak.clientSecret` key:
++
+[source,terminal]
+---- 
+$ kubectl edit secret argocd-secret -n argocd
+----
++
```

Comment:
I think the first instruction is kinda misleading, because the following steps describe how to do exactly that. So 2, 3 and 4 actually describe 1.

Reply:
Thanks. I've reformatted the step a little bit.

Reply:
:+1: looks good now!